dbfit-java/core/src/main/java/dbfit/util/DbParameterAccessorsMapBuilder.java

```diff
@@ -2,6 +2,7 @@
 
 import static dbfit.util.Direction.*;
 import static dbfit.util.NameNormaliser.*;
+import dbfit.util.TypeTransformer;
```

Comment:
Are we using this `TypeTransformer` import here, we're using the factory only as far as I see? (We're anyway in util package so we should be able to see util classes /type transformer factory/ without explicit import).